lib/services/authenticate.dart

```diff
@@ -11,21 +12,27 @@ class FireStoreUtils {
   static Reference storage = FirebaseStorage.instance.ref();
 
   static Future<User?> getCurrentUser(String uid) async {
+    log.logger.d("Getting current user by its id: $uid");
     DocumentSnapshot<Map<String, dynamic>> userDocument =
         await firestore.collection(usersCollection).doc(uid).get();
     if (userDocument.data() != null && userDocument.exists) {
-      return User.fromJson(userDocument.data()!);
+      User user = User.fromJson(userDocument.data()!);
+      log.logger.i("User exists: $user");
+      return user;
     } else {
+      log.logger.i("User does not exists with id: $uid");
```

Comment:
Is this only an "info" event?

Reply:
yeah, I usually consider application errors (like exceptions) as a warning-like event. this is just an info that the user does not exists